course/content.py

```diff
@@ -594,13 +613,19 @@ class FlowDesc(Struct):
         A list of :ref:`pages <flow-page>`. If you specify this, a single
         :class:`FlowPageGroupDesc` will be implicitly created. Exactly one of
         :attr:`groups` or :class:`pages` must be given.
+
+    .. attribute:: external_resources
+
+        A list of :class:`TabDesc`. These are links to external
+        resources that are displayed on the bottom of the flow page.
```

Comment:
No longer at the bottom. (Also contradicted by name `TabDesc`.)

Reply:
What does "contradicted by name TabDesc" refer to?